### PR TITLE
Add comments for common directory

### PR DESCRIPTION
**Pull request by watermelo**
**What this PR does**:
Add comments

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: common/config/environment.go [hunk 1]
```diff
@@ -46,15 +46,15 @@ var (
 	once     sync.Once
 )
 
-// GetEnvInstance ...
+// GetEnvInstance get env instance by singleton
```
Comment (AlexStocks) [inline common/config/environment.go:49]: get -> gets? and the following errors should be fixed too.

Comment (fangyincheng) [reply common/config/environment.go:49]: so, find should be finds, update should be updates and so no

Comment (watermelo) [reply common/config/environment.go:49]: ok

File: common/logger/logger.go [hunk 2]
```diff
@@ -125,18 +125,18 @@ func InitLogger(conf *zap.Config) {
...
 }
 
-// GetLogger ...
+// GetLogger gets the logger
 func GetLogger() Logger {
 	return logger
 }
 
-// SetLoggerLevel ...
+// SetLoggerLevel used to set logger level
```
Comment (zouyx) [inline common/logger/logger.go:139]: use for?

File: common/logger/logger.go [hunk 3]
```diff
@@ -145,13 +145,13 @@ func SetLoggerLevel(level string) bool {
 	return false
 }
 
-// OpsLogger ...
+// OpsLogger used by the SetLoggerLevel
```
Comment (zouyx) [inline common/logger/logger.go:148]: as above

File: common/logger/logger.go [hunk 4]
```diff
@@ -145,13 +145,13 @@ func SetLoggerLevel(level string) bool {
...
 type OpsLogger interface {
 	Logger
 	SetLoggerLevel(level string)
 }
 
-// SetLoggerLevel ...
+// SetLoggerLevel used to set logger level
```
Comment (zouyx) [inline common/logger/logger.go:154]: as above

File: common/node.go [hunk 5]
```diff
@@ -17,7 +17,7 @@
 
 package common
 
-// Node ...
+// Node is used to process dubbo node
```
Comment (zouyx) [inline common/node.go:20]: as above

File: common/url.go [hunk 6]
```diff
@@ -86,7 +86,7 @@ type baseUrl struct {
 	PrimitiveURL string
 }
 
-// URL ...
+// URL is used to locate resourse to transfer data between nodes
```
Comment (zouyx) [inline common/url.go:89]: use for?

File: common/url.go [hunk 7]
```diff
@@ -452,7 +454,7 @@ func (c URL) GetRawParam(key string) string {
 	}
 }
 
-// GetParamBool ...
+// GetParamBool is used to judge whether key exists or not
```
Comment (zouyx) [inline common/url.go:457]: use for?

File: common/url.go [hunk 8]
```diff
@@ -432,7 +434,7 @@ func (c URL) GetParamAndDecoded(key string) (string, error) {
 	return value, err
 }
 
-// GetRawParam ...
+// GetRawParam is used to get raw param
```
Comment (zouyx) [inline common/url.go:437]: use for?

**Comment by fangyincheng** [comment]
please resolve conflicts

File: common/config/environment.go [hunk 9]
```diff
@@ -90,17 +91,17 @@ func (env *Environment) Configuration() *list.List {
 	return cfgList
 }
 
-// SetDynamicConfiguration ...
+// SetDynamicConfiguration is used to set value for dynamicConfiguration
```
Comment (AlexStocks) [inline common/config/environment.go:94]: SetDynamicConfiguration sets value for dynamicConfiguration

File: common/config/environment.go [hunk 10]
```diff
@@ -109,7 +110,7 @@ func newInmemoryConfiguration(p *sync.Map) *InmemoryConfiguration {
 	return &InmemoryConfiguration{store: p}
 }
 
-// GetProperty ...
+// GetProperty is used the key to get value from InmemoryConfiguration instance
```
Comment (AlexStocks) [inline common/config/environment.go:113]: GetProperty gets value from InmemoryConfiguration instance by @key

File: common/extension/config_reader.go [hunk 11]
```diff
@@ -26,25 +26,25 @@ var (
...
+// GetConfigReaders gets a config reader with @name
 func GetConfigReaders(name string) interfaces.ConfigReader {
 	if configReaders[name] == nil {
 		panic("config reader for " + name + " is not existing, make sure you have imported the package.")
 	}
 	return configReaders[name]()
 }
 
-// SetDefaultConfigReader set {name} to default config reader for {module}
+// SetDefaultConfigReader sets {name} to default config reader for {module}
```
Comment (zouyx) [inline common/extension/config_reader.go:42]: what's the different `{name}` and `@name` ?

**Pull request merged**